onnx/test/checker_test.py

```diff
@@ -91,15 +91,23 @@ def test_check_function_nested(self) -> None:
             func_nested_identity_add_nodes,
             func_nested_opset_imports,
         )
-        checker.check_function(func_nested_identity_add)
+        ctx = checker.C.CheckerContext()
+        ctx.ir_version = 3
```

Comment:
Why ir_version 3? Is that intentional?

Reply:
I updated it to the default `IR_VERSION` from onnx import.